zerver/actions/custom_profile_fields.py

```diff
@@ -108,7 +119,9 @@ def try_update_realm_custom_profile_field(
     required: bool = False,
 ) -> None:
     field.name = name
+    field.rendered_name = render_custom_profile_fields(name, realm)
     field.hint = hint
+    field.rendered_hint = render_custom_profile_fields(hint, realm)
```

Comment:
Would be worth adding a small test verifying the `rendered_...` fields on creating and updating custom profile fields.

Reply:
added!